packages/storage-plus/src/snapshot/map.rs

```diff
@@ -262,6 +262,33 @@ where
     }
 }
 
+#[cfg(feature = "iterator")]
+impl<'a, K, T> SnapshotMap<'a, K, T>
+where
+    T: Serialize + DeserializeOwned + Clone,
+    K: PrimaryKey<'a> + Prefixer<'a> + KeyDeserialize,
+{
+    pub fn changelog_range<'c>(
```

Comment:
I think this approach is in the right direction, but not quite there yet.

What I recommend is to generalize this, i.e. instead of providing an ad-hoc `range` method that does what you need, provide instead the entire set of functions, both for compatibility and completeness.